operations/mimir-mixin/docs/playbooks.md

```diff
@@ -195,6 +195,10 @@ How to **investigate**:
 - Check the `Mimir / Reads` dashboard
   - Looking at the dashboard you should see in which Mimir service the high latency originates
   - The panels in the dashboard are vertically sorted by the network path (eg. gateway -> query-frontend -> query->scheduler -> querier -> store-gateway)
+- Check the `Mimir / Reads resources` dashboard
+  - Check if query frontend pods are at the memory limit - if they are, check in Kubernetes if pods are OOM-ing
```

Comment:
Also, please spell it "query-frontend".

Reply:
> This runbook is about "Read latency". If query-frontend OOMs, then the impact is not higher read latency, but higher failure rate on read path.

@pracucci I didn't notice higher failure rate while query-frontend replicas were OOM-ing, are you sure it will consistently translate to errors? It is possible I failed to look for failed requests, but I checked for them initially at least (and deduced latency was the problem).